metricshub-doc/src/site/markdown/connector-design-guide/presources.md

```diff
@@ -0,0 +1,20 @@
+keywords: guide, connector, pre, presources
+description:This page define sources that need to be executed before each monitoring job (discovery, multiCollect, …).
+
+# Pre Sources
+
+This page define sources that need to be executed before each monitoring job (discovery, multiCollect, …).

```

Comment:
This page outlines the sources that must be run prior to each monitoring job (such as discovery, collect, simple etc.). These preliminary sources can be referenced in other sources within a monitoring job.